bigtable/client/internal/port_platform.h

```diff
@@ -68,4 +68,10 @@
 #endif  // GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
 // clang-format on
 
+#ifndef GOOGLE_CLOUD_CPP_DISABLE_PRIVATE_ALPHA_WARNINGS
+#define GOOGLE_CLOUD_CPP_PRIVATE_ALPHA [[deprecated("private alpha release")]]
```

Comment:
Curious why alpha features are marked as "deprecated"? Is `@warning` not sufficient for this? Will the user see the word "deprecated" in the output, or just see the word "warning"?

Reply:
The only C++ annotation that creates a warnings is the `[[deprecated]]` annotation.  It is ugly, and it is one of the reasons I wanted you and @ghaisa to take a look.

Reply:
Will the literal word "deprecated" appear anywhere in the output (e.g., compiler-generated warning messages), or will it mark these APIs in any way as "deprecated" anywhere (e.g., IDE) other than that C++ macro? Or will users only see the literal warning that you add in the macro after the `deprecated` keyword?